src/main/webapp/app/shared/grading-instruction-link-icon/grading-instruction-link-icon.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+import { GradingInstruction } from 'app/exercises/shared/structured-grading-criterion/grading-instruction.model';
+import { Feedback } from 'app/entities/feedback.model';
+
+@Component({
+    selector: 'jhi-grading-instruction-link-icon',
+    templateUrl: './grading-instruction-link-icon.component.html',
+})
+export class GradingInstructionLinkIconComponent implements OnInit {
+    @Input() linkIcon = <IconProp>'link';
+    @Input() assessment: Feedback;
+    instruction: GradingInstruction | undefined;
+    confirmIcon = <IconProp>'trash';
+    showConfirm = false;
+
+    ngOnInit(): void {
+        this.instruction = this.assessment.gradingInstruction;
+    }
+
+    /**
+     * remove grading instruction on click
+     */
+    removeLink(): void {
+        this.toggle();
+        this.assessment.gradingInstruction = undefined;
+    }
+
+    /**
+     * Set the tooltip of the link icon to be equal to the grading instruction description text
+     * @param {GradingInstruction} instruction - the instruction object which is associated with feedback
+     */
+    setTooltip(instruction: GradingInstruction) {
+        return 'Grading Instruction: ' + instruction.instructionDescription;
```

Comment:
This one should also be part of the translation file, I think it would also be useful to state that clicking it allows to delete the link; I don't think many would randomly click on it